spec/presenters/api/v2/rules_of_origin/scheme_presenter_spec.rb

```diff
@@ -0,0 +1,26 @@
+RSpec.describe Api::V2::RulesOfOrigin::SchemePresenter do
+  subject(:presenter) { described_class.new scheme, rules }
+
+  let(:scheme) { build :rules_of_origin_scheme }
+
+  let(:rules) do
+    build_list :rules_of_origin_rule, 3, scheme_code: scheme.scheme_code
+  end
+
+  it { is_expected.to be_instance_of described_class }
```

Comment:
this sounds redundant (the described_class could be any class and this would be always true), we could remove it, up to you anyway 😄

Reply:
Yeh, I know it looks unnecessary - its a stupid trivial test, but also very fast - I don't really care about the response but I want a very simple check to make sure the initializer is working correctly - all the other tests could fail because of different issues but this one can only fail if the initializer is not working right.